hawkular-inventory-api/src/test/java/org/hawkular/inventory/api/test/AbstractBaseInventoryTestsuite.java

```diff
@@ -2554,21 +2569,21 @@ public void testObserveIdentityHashChangedOnParentOfDeletedEntity() throws Excep
         });
     }
 
-    private void testIdentityHashObservation(String tenantId, Consumer<Map<Class<?>, Integer>> test) throws Exception {
+    private void testSyncHashObservation(String tenantId, Consumer<Map<Class<?>, Integer>> test) throws Exception {
         Subscription subs = null;
         try {
-            createInventoryForIdentityHashChangeObservations(tenantId);
+            createInventoryForSyncHashChangeObservations(tenantId);
 
             Map<Class<?>, Integer> updateCounts = new HashMap<>();
 
             subs = Observable.merge(
-                    inventory.observable(Interest.in(Feed.class).having(identityHashChanged())),
-                    inventory.observable(Interest.in(ResourceType.class).having(identityHashChanged())),
-                    inventory.observable(Interest.in(MetricType.class).having(identityHashChanged())),
-                    inventory.observable(Interest.in(OperationType.class).having(identityHashChanged())),
-                    inventory.observable(Interest.in(Resource.class).having(identityHashChanged())),
-                    inventory.observable(Interest.in(Metric.class).having(identityHashChanged())),
-                    inventory.observable(Interest.in(DataEntity.class).having(identityHashChanged()))
+                    inventory.observable(Interest.in(Feed.class).having(syncHashChanged())),
+                    inventory.observable(Interest.in(ResourceType.class).having(syncHashChanged())),
+                    inventory.observable(Interest.in(MetricType.class).having(syncHashChanged())),
+                    inventory.observable(Interest.in(OperationType.class).having(syncHashChanged())),
+                    inventory.observable(Interest.in(Resource.class).having(syncHashChanged())),
+                    inventory.observable(Interest.in(Metric.class).having(syncHashChanged())),
+                    inventory.observable(Interest.in(DataEntity.class).having(syncHashChanged()))
```

Comment:
The intention is that anyone - either glue or agent can react on changes. They can react on content hash change if they want to react on the entity itself changing in some way (probably this is identical to reacting to update). They can react on identity hash change if they want to know if the entity itself or any of its children has changed structurally, or they can react on sync hash change if they want to react on the entity or any of its children being changed in any way.